chapters/license-matching-guidelines-and-templates.md

```diff
@@ -12,13 +12,15 @@ To ensure consistent results by different SPDX document creators when matching l
 
 ### B.2.2 Guideline: official license headers <a name="B.2.2"></a>
 
-The same matching guidelines used for license and exception text apply to the official license headers. Where applicable, an official license header template file with markup is included with the SPDX License List. Official license headers are defined by the SPDX License List as specific text specified within the license itself to be put in the header of files. (see https://spdx.org/spdx-license-list/license-list-overview for more info).
+The matching guidelines apply to license and exception text, as well as  official license headers. Official license headers are defined by the SPDX License List as specific text specified within the license itself to be put in the header of files. (see [explanation of SPDX License List fields](https://github.com/spdx/license-list-XML/blob/master/DOCS/license-fields.md) for more info).
+
+The following XML tag is used to implement this guideline: `<standardLicenseHeader>`
 
 ## B.3 Substantive text <a name="B.3"></a>
 
 ### B.3.1 Purpose <a name="B.3.1"></a>
 
-To ensure that when matching licenses and exceptions to the SPDX License List, there is an appropriate balance between matching against the substantive text and disregarding parts of the text that do not alter the substantive text. Further guidelines of what can be disregarded or considered replaceable for purposes of matching are listed below here and in the subsequent specific guidelines. A conservative approach is taken in regard to rules about disregarded or replaceable text.
+To ensure that when matching licenses and exceptions to the SPDX License List, there is an appropriate balance between matching against the substantive text and disregarding parts of the text that do not alter the substantive text or legal meaning. Further guidelines of what can be disregarded or considered replaceable for purposes of matching are listed below here and in the subsequent specific guidelines. A conservative approach is taken in regard to rules disregarded or replaceable text.
```

Comment:
I think the word "about" was incorrectly removed from the last sentence? I think the word "disregarded" is meant to refer to "disregarded text", so "about" should probably be reinserted.

Reply:
added " rules relating to"